vic/vic_run/src/lakes.eb.c

```diff
@@ -26,6 +26,7 @@
 
 #include <vic_def.h>
 #include <vic_run.h>
+#include <vic_driver_shared.h>
```

Comment:
Actually, this is because the lake module needs the initialize_lake() function (which is currently in the driver section).  When a lake shrinks to nothing, and then re-appears, the state of the new lake needs to be initialized - that's why the lake module needs the function.

We could move initialize_lake() into vic_run/ if that would help.  vic_init() is already including vic_run.h (which is where we'd put the definition of initialize_lake()).